packages/ui/react-ui/src/CrossMintButtonStatus.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { CSSProperties, FC, MouseEvent, MouseEventHandler, useMemo, useCallback } from "react";
+import useCrossMintStatus, { OnboardingRequestStatusResponse } from "./hooks/useCrossMintStatus";
+import { Button, Img, Paragraph } from "./styles/index";
+
+export interface StatusButtonProps {
+    className?: string;
+    disabled?: boolean;
+    onClick?: (e: MouseEvent<HTMLButtonElement>) => void;
+    style?: CSSProperties;
+    tabIndex?: number;
+    clientId: string;
```

Comment:
Same as with `CrossmintButton`, now both `clientId` and `auctionId` props need to be provided in the button.